src/StoreKeeper/WooCommerce/B2C/Endpoints/Webhooks/InfoHandler.php

```diff
@@ -98,6 +104,124 @@ public static function getExtras()
         return $extras;
     }
 
+    public static function getSystemStatus(): array
+    {
+        $systemStatus = [];
+        $systemStatus['order'] = self::getOrderSystemStatus();
+        $systemStatus['task_processor'] = self::getTaskProcessorStatus();
+        $systemStatus['failed_compatibility_checks'] = self::getFailedCompatibilityChecks();
+
+        return $systemStatus;
+    }
+
+    public static function getFailedCompatibilityChecks(): array
+    {
+        return array_merge(
+            PluginConflictChecker::getPluginsWithConflict(),
+            ServerStatusChecker::getServerIssues(),
+        );
+    }
+
+    public static function getTaskProcessorStatus(): array
+    {
+        $taskProcessorStatus = [];
+
+        $now = current_time('mysql', 1);
+        $calculator = new TaskRateCalculator($now);
+        $processedRate = $calculator->calculateProcessed();
+        $cronRunner = CronOptions::get(CronOptions::RUNNER, CronRegistrar::RUNNER_WPCRON);
+        $postExecutionStatus = CronOptions::get(CronOptions::LAST_EXECUTION_STATUS, CronRegistrar::STATUS_UNEXECUTED);
+        $postExecutionError = CronOptions::get(CronOptions::LAST_POST_EXECUTION_ERROR);
+        $preExecutionDateTime = CronOptions::get(CronOptions::LAST_PRE_EXECUTION_DATE);
+        $invalidRunners = CronOptions::getInvalidRunners();
+
+        $postExecutionSuccessful = CronRegistrar::STATUS_SUCCESS === $postExecutionStatus;
+
+        $lastProcessTaskDate = TaskModel::getLastProcessTaskDate();
+        $lastProcessTaskDateFormatted = null;
+        if ($lastProcessTaskDate) {
+            $lastProcessTaskDateFormatted = \DateTime::createFromFormat('Y-m-d H:i:s', $lastProcessTaskDate)->format(DATE_RFC2822);
```

Comment:
```suggestion
            $lastProcessTaskDateFormatted = \DateTime::createFromFormat('Y-m-d H:i:s', $lastProcessTaskDate)->format(DATE_RFC2822);
```

Hmm we do not have a const for the format `'Y-m-d H:i:s'` ? If this date stored as UTC date or in lokal timezone?

Reply:
Aadded changes for implementing uniform format of database dates.